packages/elemental-theme/src/custom-elements/ef-autosuggest.less

```diff
@@ -52,7 +53,7 @@
 
   [part=backdrop] {
     background-color: @overlay-background-color;
-    opacity: @overlay-opacity;
```

Comment:
Why did you override this?

Reply:
It is wrong all together: being the loader too large and the background too grey. Perhaps we need a proper solution from Design/UX on how to cover this case (and all other cases with asynchronous popup loading). This code was just migrated from v3 where it made more sense.

Reply:
maybe UX can say that entire screen needs to be greyed out, so we can deprecated autosuggest loading :)

Reply:
I'll go with this small improvement. It's better than today anyway. You know we would need to wait very long until they take an action. It means that users won't see any improvement until then :)

Reply:
Updated!, I have revert override backdrop opacity :)